src/get-started/webpack.md

```diff
@@ -1,8 +1,45 @@
 ---
 layout: page
-title: Get Started > Webpack
+title: Get Started > webpack
 ---
 
-# Webpack
+# webpack
 
-Need to write up.
+The following example uses [workbox-webpack-plugin](https://www.npmjs.com/package/workbox-webpack-plugin)
+to create a precaching service worker in your webpack build process. This will
+give you a new service worker every time you run it.
+
+1. [Install Node.js](https://nodejs.org/en/).
```

Comment:
It's useful to look at how other Webpack plugins handle the install flow: https://github.com/jantimon/html-webpack-plugin/. They assume both Node and Webpack are already installed and dive straight-into adding in the plugin.